lib/exception_hunter/devise.rb

```diff
@@ -0,0 +1,22 @@
+require 'devise'
```

Comment:
What happens if the user does not have devise installed? This would throw an error. We need to find a way to require devise only if the user has installed it